can-define.js

```diff
@@ -288,6 +288,7 @@ make = {
 		events: function(prop, getCurrent, setData, eventType) {
 			return function(newVal) {
 				if (this.__inSetup) {
+					Observation.ignore(getCurrent).call(this);
```

Comment:
do you need the `Observation.ignore`? It looks like `getCurrent` only adds an observation when not `__inSetup`.